astropy/convolution/boundary_extend.pyx

```diff
@@ -102,7 +102,7 @@ def convolve2d_boundary_extend(np.ndarray[DTYPE_t, ndim=2] f,
     cdef int wky = nky // 2
```

Comment:
Actually I take it back, I didn't realize that `ii` and `jj` can be negative until now.  Curse these poor variable names.